src/ir_Mitsubishi.h

```diff
@@ -277,14 +287,27 @@ class IRMitsubishiAC {
   uint8_t getFan(void) const;
   void setMode(const uint8_t mode);
   uint8_t getMode(void) const;
-  bool getiSave10C(void) const;    //1 = in i-Save mode and temp = 10C
-  void setiSave10C(bool);          //1 = set to i-Save mode when temp = 10C on remote. "Temp" byte is not affected by this(!). 10C temp is implied through other bits
-  void setVane(const uint8_t position);
-  void setWideVane(const uint8_t position);
+  /// 1 = set to i-Save when temp=10C on remote.
+  /// "Temp" byte is not affected by this; 
+  /// 10C temp is implied through other bits.
```

Comment:
Move those comments in/near the procedures in the `.cpp` file. e.g. Doxygen will automatically turn the comments into documentation for us.